src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestEngine.cs

```diff
@@ -0,0 +1,105 @@
+            var saveCdkDirectoryRelativePath = _directoryManager.GetRelativePath(TargetApplicationFullPath, saveCdkDirectoryFullPath);
+
+            DeploymentManifestModel deploymentManifestModel;
+
+            if (_fileManager.Exists(deploymentManifestFilePath))
+            {
+                deploymentManifestModel = await ReadManifestFile(deploymentManifestFilePath);
+                deploymentManifestModel.DeploymentManifestEntries.Add(new DeploymentManifestEntry(saveCdkDirectoryRelativePath));
+            }
+            else
+            {
+                var deploymentManifestEntries = new List<DeploymentManifestEntry> { new DeploymentManifestEntry(saveCdkDirectoryRelativePath) };
+                deploymentManifestModel = new DeploymentManifestModel(deploymentManifestEntries);
+            }
+
+            var manifestFileJsonString = SerializeManifestModel(deploymentManifestModel);
+            await _fileManager.WriteAllTextAsync(deploymentManifestFilePath, manifestFileJsonString);
+        }
+
+        /// <summary>
+        /// This method parses the deployment-manifest file into a <see cref="DeploymentManifestModel"/>
+        /// </summary>
+        /// <param name="filePath">The path to the deployment-manifest file</param>
+        /// <returns>An instance of <see cref="DeploymentManifestModel"/></returns>
+        private async Task<DeploymentManifestModel> ReadManifestFile(string filePath)
+        {
+            var manifestFilejsonString = await _fileManager.ReadAllTextAsync(filePath);
+            return JsonConvert.DeserializeObject<DeploymentManifestModel>(manifestFilejsonString);
+        }
+
+        /// <summary>
+        /// This method parses the <see cref="DeploymentManifestModel"/> into a string
+        /// </summary>
+        /// <param name="deploymentManifestModel"><see cref="DeploymentManifestModel"/></param>
+        /// <returns>A formatted string representation of <see cref="DeploymentManifestModel"></returns>
+        private string SerializeManifestModel(DeploymentManifestModel deploymentManifestModel)
+        {
+            return JsonConvert.SerializeObject(deploymentManifestModel, Formatting.Indented);
+        }
+
+        /// <summary>
+        /// This method returns the path at which the deployment-manifest file will be stored.
+        /// </summary>
+        /// <returns>The path to the deployment-manifest file.</returns>
+        private string GetDeploymentManifestFilePath()
+        {
+            if (string.IsNullOrEmpty(TargetApplicationFullPath))
+                throw new InvalidOperationException($"{nameof(TargetApplicationFullPath)} is null as part of the " +
+                    $"{nameof(DeploymentManifestEngine)} object");
+
+            var projectDirectoryFullPath = _directoryManager.GetDirectoryInfo(TargetApplicationFullPath).Parent.FullName;
+            var deploymentManifestFileFullPath = Path.Combine(projectDirectoryFullPath, "aws-deployments.json");
```

Comment:
define the file name as a private constant and refer to it here.

Reply:
Fixed